src/js/components/Drop/__tests__/Drop-test.tsx

```diff
@@ -282,6 +315,34 @@ describe('Drop', () => {
     });
   });
 
+  test('focus does not leave the dialog when trapFocus is true', async () => {
+    userEvent.setup();
+    window.scrollTo = jest.fn();
+    render(<TestTrapFocus />);
+
+    // Wait for the button with text 'first-focus'
+    const firstButton = await screen.findByText('first-focus');
+    const secondButton = await screen.findByText('second-focus');
+    expect(firstButton).toBeInTheDocument();
+
+    await userEvent.tab();
+
+    // Check that the button is the currently focused element
+    expect(document.activeElement).toBe(firstButton);
+
+    // Simulate tab
+    await userEvent.tab();
+
+    // Expect the focus to go to the second button
+    expect(document.activeElement).toBe(secondButton);
+
+    // Simulate tab
+    await userEvent.tab();
+
+    // Expect the focus to go back to the first button
+    expect(document.activeElement).toBe(firstButton);
```

Comment:
is there a way to include 
```
await userEvent.tab({ shift: true });

```
does that work just to check if after its then on the secondButton 

not super high need but would be nice to check

Reply:
Good idea, added this in latest commit